workflow_imaging/ingest.py

```diff
@@ -57,9 +60,9 @@ def ingest():
                                   skip_duplicates=True)
 
     # caiman - 3D
-    imaging.ProcessingTask.insert([{**sc, 'paramset_idx': 2, 'task_mode': 'load'}
-                                   for sc in (scan.Scan & (scan.ScanInfo & 'ndepths > 1')).fetch('KEY')],
-                                  skip_duplicates=True)
+    # imaging.ProcessingTask.insert([{**sc, 'paramset_idx': 2, 'task_mode': 'load'}
```

Comment:
Did you mean to comment out? Please remove code that's commented out.

Reply:
this is commented out temporarily in the `dev` branch, we will add a new set of parameters for CaImAn 3d and uncomment this block